e2e/test/scenarios/question/nested.cy.spec.js

```diff
@@ -52,8 +52,6 @@ describe("scenarios > question > nested", () => {
       name: "GH_12568: Simple",
       query: {
         "source-table": ORDERS_ID,
-        aggregation: [["count"]],
-        breakout: [["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }]],
```

Comment:
_What this was supposed to do_. This test checks whether it is possible to do _sum over time_ drill on __aggregating query__. In this case sum over time does summation by already existing breakout from subquery. That results in no change to data displayed in table view.

_Why that is incompatible with this PR_. This PR changes modifies `available-temporal-buckets` in a way, that for columns that are already bucketed, the returned `:default` bucket is none. I believe that if bucketing was done in previous stage I don't want to do it in the next stage by default. This is relevant eg. in QB's breakout step. So eg. when on previous stage `created_at` was bucketed by year, on this stage QB won't prefill `:month` (the original default) for the column.

_What is this change supposed to achieve_. Test is modified so it checks whether drill works on unbucketed column from nested query, which has a `:month` default.